adback-solution-to-adblock/admin/partials/ad-back-admin-iab-banners-display.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
I think this file should be suppressed as it as been replaced with the placement  one